### DISCUSSION TITLE
json-server 배포 : Heroku

### DISCUSSION THREAD

**Discussion by hyoniiii** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#34] [Reference]
과제 내용을 살펴보면서 api를 로컬에서 돌리면 배포시에 어떻게 연결을 해야하나 고민 하던 와중에 Heroku라는 클라우드 컴퓨팅 플랫폼(PaaS)을 찾았습니다.

사용 방법 : package.json 설정 변경 / Heroku CLI 설치 -> git push -> Heroku에 업로드

참고 : https://redux-advanced.vlpt.us/3/09.html

앗 근데 **11월 28**일까지만 무료플랜을 제공한다고 하네요...!

**Comment by jaeyeong815** [comment]
헉 저 heroku로 배포해둔게 두개 있는데.. 도르륵.. 28일까지만 무료면.. 옮겨야하네요..
